tensorflow/python/ops/sparse_ops.py

```diff
@@ -2732,6 +2733,99 @@ def sparse_transpose(sp_input, perm=None, name=None):
     return transposed_st
 
 
+@tf_export("sparse.map_values")
+@dispatch.add_dispatch_support
+def map_values(op, *args, **kwargs):
+  """Applies `op` to the values of one or more `SparseTensor`s.
+
+  Replaces any `SparseTensor` in `args` or `kwargs` with its `values`
+  tensor, and then calls `op`.  Returns a `SparseTensor` that is constructed
+  from the input `SparseTensor`s' `indices` and the value returned by
+  the `op`.
+
+  If the input arguments contain multiple `SparseTensor`s, then they must have
+  identical `indices`.
+
+  Examples:
+
+  >>> st = tf.sparse.from_dense([[1, 2, 0], [0, 4, 0], [1, 0, 0]])
+  >>> tf.sparse.to_dense(tf.sparse.map_values(tf.ones_like, st)).numpy().to_list()
+  [[1, 1, 0], [0, 1, 0], [1, 0, 0]]
+  >>> tf.sparse.to_dense(tf.sparse.map_values(tf.multiply, st, st)).numpy().to_list()
+  [[1, 4, 0], [0, 16, 0], [1, 0, 0]]
+  >>> tf.sparse.to_dense(tf.sparse.map_values(tf.add, st, 5)).numpy().to_list()
+  [[5, 7, 0], [0, 9, 0], [6, 0, 0]]
+
+  Note in particular that even though `tf.add(0, 5) != 0`, implicit zeros
+  will remain unchanged. However, if the sparse tensor contains any explict
+  zeros, these will be affected by the mapping!
+
+  Args:
+    op: The operation that should be applied to the SparseTensor `values`.
+      `op` is typically an element-wise operation (such as math_ops.add), but
+      any operation that preserves the shape can be used.
+    *args: Arguments for `op`.
+    **kwargs: Keyword arguments for `op`.
+
+  Returns:
+    A `SparseTensor` whose `indices` matches the `indices` of all
+    input `SparseTensor`s.
+  Raises:
+    ValueError: If args contains no `SparseTensor`, or if the `indices`
+      of the input `SparseTensor`s are not identical.
```

Comment:
Maybe "not equal" instead of "not identical" -- the latter could be read to mean that they need to be the same tensor.